src/validation/helpers/writeInfoInline.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Writes information about if a type is required and if it can be empty inline
+ */
+export default function writeInfoInline(type, canBeEmpty, required) {
+    const empty = canBeEmpty ? '?' : '';
+    return type && `${required ? `<${empty}${type}>` : `[${empty}${type}]`}`;
```

Comment:
Do we want to force this to always return a string? Right now it'll return a falsy value or a string